build/includes/website.mk

```diff
@@ -75,6 +75,7 @@ site-test:
 
 # generate site images, if they don't exist
 site-images: $(site_path)/static/diagrams/gameserver-states.dot.png $(site_path)/static/diagrams/gameserver-lifecycle.puml.png $(site_path)/static/diagrams/gameserver-reserved.puml.png
```

Comment:
nit: It would be nice to break this super long line into multiple lines with one image / line to make this easier to read.

Reply:
I like it. Also less merge conflicts.